lib/diagnostic.js

```diff
@@ -19,15 +19,15 @@
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = False; // replace null with your response
```

Comment:
this would be true because between line 7 and 8 is still within the scope of deathstar